tests/cache/models/redisTestUtil.js

```diff
@@ -115,6 +119,17 @@ module.exports = {
     .catch(done);
   },
 
+  forceDelete(done) {
+    tu.forceDelete(tu.db.Sample, testStartTime)
+    .then(() => tu.forceDelete(tu.db.Aspect, testStartTime))
```

Comment:
why it is twice ?

Reply:
Keeping this because it is used by other tests.